assignment-1/annabel-chavez.sql

```diff
@@ -0,0 +1,74 @@
+
+Question 1
+
+SELECT id, user_id, completed_at, charge_amount FROM fact_order
+WHERE charge_amount > 500 AND completed_at is not null
+ORDER BY  completed_at desc
+LIMIT 1
+
+-------------------------------
+
+Question 2
+
+SELECT date_trunc('week',completed_at) AS completed_week, SUM(f.charge_amount) AS total_gmv, count(f.id) AS order_count 
+FROM fact_order f
+    JOIN zones z on f.zone_id = z.id
+    JOIN regions r on z.region_id = r.id
+WHERE r.country_id = 124
+AND completed_week BETWEEN '2019-09-23' AND '2019-11-18'
+GROUP BY 1
+ORDER BY 1
```

Comment:
Instead of that, we could improve this line with `completed_week DESC` in order to better display the results:
```suggestion
ORDER BY completed_week DESC
```